caddy.go

```diff
@@ -182,7 +182,7 @@ func (i *Instance) Restart(newCaddyfile Input) (*Instance, error) {
 	newInst := &Instance{serverType: newCaddyfile.ServerType(), wg: i.wg}
 
 	// attempt to start new instance
-	err := startWithListenerFds(newCaddyfile, newInst, restartFds)
+	err := startWithListenerFds(newCaddyfile, newInst, restartFds, false)
```

Comment:
I assume there will never be a scenario where justValidate is expected to be true on a restart